src/c75_menuCTRL.hpp

```diff
@@ -111,7 +115,98 @@ bool processControlKeys()
                     ctrlState = MANUAL_CONTROL_MODE;
                     mount.stopSlewing(ALL_DIRECTIONS);
                 }
-                else if ((key == btnDOWN) || (key == btnUP))
+                else if (key == btnDOWN)
+                {
+        #if USE_HALL_SENSOR_RA_AUTOHOME == 1
+                    ctrlState = HIGHLIGHT_RA_AUTO_HOME;
+        #else
+                    ctrlState = HIGHLIGHT_SERIAL;
+        #endif
+                }
+                //DEC-
+                else if (key == btnDOWN)
```

Comment:
This will never execute

Reply:
I have posted a video showing that it works...